taskcluster/ci/push-bundle/kind.yml

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.from_deps
+    - android_taskgraph.transforms.upstream_artifacts
+    - android_taskgraph.transforms.treeherder
+    - android_taskgraph.transforms.push_android_app
+    - taskgraph.transforms.task
+
+kind-dependencies:
+    - signing-bundle
+    - startup-test
+
+tasks:
+    push-bundle:
+        description: Publish Fenix AABs
+        from-deps:
+            with-attributes:
+                build-type:
+                    - fenix-nightly
+                    # coming soon!
+                    # - fenix-beta
+                    # - fenix-release
+            group-by:
+                attribute: build-type
+            copy-attributes: true
+        worker-type: push-apk
+        run-on-tasks-for: []
+        worker:
+            certificate-alias:
+                by-build-type:
+                    fenix-nightly: fenix-nightly
+                    fenix-beta: fenix-beta
+                    fenix-release: fenix-release
+            commit: true
+            channel:
+                by-build-type:
+                    fenix-nightly: fenix-nightly
+                    fenix-beta: fenix-beta
+                    fenix-release: fenix-release
+            dep:
+                by-level:
+                    '3': false
+                    default: true
+            product: fenix
+        shipping-phase: push
+        treeherder:
+            job-symbol: gp-aab
+            kind: build
+        routes:
+            by-level:
+                '3':
+                  - "notify.email.release-mgmt@mozilla.com.on-failed"
+                'default': []
```

Comment:
Nit: If I'm not mistaken, this config is almost identical to `push-apk/kind.yml`. I'd recommend we centralize this in a transforms file or maybe use the new `task-context` @bhearsum implemented[1].

That said, I know this file is going to change again before Fenix release is shipped so I won't block on that. 

[1] https://mozilla.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=d657c9ba-ecf8-4c94-a875-b08a0125b9ca&start=420.220708

Reply:
Thanks. I have thought about that. In the coming weeks, as we test and roll out Fenix AAB, I think it will be less confusing (for me anyway!) to keep separate apk and aab kinds. We can revisit and optimize once the changes have stabilized.